source/js/components/newsletter-signup/atoms/description.jsx

```diff
@@ -1,12 +1,22 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const Description = ({ children }) => {
-  return <p>{children}</p>;
+const Description = ({ content }) => {
+  if (typeof content === "string") {
+    return (
+      <div
+        dangerouslySetInnerHTML={{
+          __html: content,
+        }}
+      />
+    );
+  }
```

Comment:
Is this a safeguard against `description` returning a string? If so, I think it will always be enclosed in a `div` with a `rich-text` class.

Reply:
Thanks for flagging this, @jhonatan-lopes 

We have some existing use cases where values of `description` and `header are not coming from the CMS but directly from the template (e.g., hardcoded). Having the `type` check here ensures this React component can work for all cases.